### PR TITLE
[zh] fix translation errors

### PR DESCRIPTION
**Pull request by wulianglongrd**
Please provide a description for what this PR is for.

fix translation errors

english version:
> For request authentication, the application is responsible for acquiring and attaching the JWT credential to the request. 
> For peer authentication, Istio automatically upgrades all traffic between two PEPs to mutual TLS.

current(incorrect):
> 对于 peer authentication，应用程序负责获取 JWT 凭证并将其附加到请求。
> 对于双向 TLS，Istio 会自动将两个 PEPs 之间的所有流量升级为双向 TLS。

this PR(correct):
> 对于 request 认证，应用程序负责获取 JWT 凭证并将其附加到请求。
> 对于 peer 认证，Istio 自动将两个 PEP 之间的所有流量升级为双向 TLS。

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Review by windsonsea — COMMENTED** [review]
authentication 身份验证，认证

如果这里要改成`身份验证`，那这个页面可能需要修改 67 处。
需要逐一确认 en 是否为 authentication

File: content/zh/docs/concepts/security/index.md [hunk 1]
```diff
@@ -161,7 +161,7 @@ Istio 双向 TLS 具有一个宽容模式（permissive mode），允许服务同
 
 Istio 异步发送配置到目标端点。代理收到配置后，新的认证要求会立即生效。
 
-发送请求的客户端服务负责遵循必要的认证机制。对于 peer authentication，应用程序负责获取 JWT 凭证并将其附加到请求。对于双向 TLS，Istio 会自动将两个 PEPs 之间的所有流量升级为双向 TLS。如果认证策略禁用了双向 TLS 模式，则 Istio 将继续在 PEPs 之间使用纯文本。要覆盖此行为，请使用 [destination rules](/zh/docs/concepts/traffic-management/#destination-rules)显式禁用双向 TLS 模式。您可以在[双向 TLS 认证](/zh/docs/concepts/security/#mutual-TLS-authentication) 中找到有关双向 TLS 如何工作的更多信息。
+发送请求的客户端服务负责遵循必要的身份验证机制。对于 request authentication，应用程序负责获取 JWT 凭证并将其附加到请求。对于 peer authentication，Istio 会自动将两个 PEPs 之间的所有流量升级为双向 TLS。如果身份验证策略禁用双向 TLS 模式，Istio 将继续在 PEPs 之间使用纯文本。要覆盖此行为，请使用 [destination rules](/zh/docs/concepts/traffic-management/#destination-rules)显式禁用双向 TLS 模式。您可以在[双向 TLS 认证](/zh/docs/concepts/security/#mutual-TLS-authentication) 中找到有关双向 TLS 如何工作的更多信息。
```
Comment (windsonsea) [inline content/zh/docs/concepts/security/index.md:164]:
```suggestion
发送请求的客户端服务负责遵循必要的身份验证机制。对于 request authentication，应用程序负责获取 JWT 凭证并将其附加到请求。对于 peer authentication，Istio 会自动将两个 PEP 之间的所有流量升级为双向 TLS。如果身份验证策略禁用双向 TLS 模式，Istio 将继续在 PEP 之间使用纯文本。要覆盖此行为，请使用 [destination rules](/zh/docs/concepts/traffic-management/#destination-rules)显式禁用双向 TLS 模式。您可以在[双向 TLS 认证](/zh/docs/concepts/security/#mutual-TLS-authentication) 中找到有关双向 TLS 如何工作的更多信息。
```
另外这 2 个词组如果不是指代码，翻译了比较好：

- request authentication
- peer authentication

image

**Comment by windsonsea** [comment]
整篇文章 67 处 `authentication 认证`，只改这一句不可取。

**Comment by wulianglongrd** [comment]
@windsonsea What I did was fix the translation error (see PR description), not improve the terminology (you said "认证" -> "验证"), but you said it was indeed a problem, and now, that's fixed.

**Review by windsonsea — APPROVED** [review]

**Pull request merged**